docs/user/exams/students_guide.rst

```diff
@@ -25,11 +25,11 @@ Prerequisites
 
     - **Java IDE with JDK 17**
 
-        **Recommendation:** Eclipse IDE 2021‑12.
+        **Recommendation:** IntelliJ IDEA 2022.1.3.
```

Comment:
Why do you want to drop the recommendation for eclipse?
I would suggest to simply add the recommendation for intellij here.

Reply:
```suggestion
        **Recommendation:** IntelliJ IDEA 2022.1.3 or Eclipse IDE 2022-06
```

Reply:
We completely switched to IntelliJ with Gradle in our course. It's much easier and more reliable to use than Eclipse ;-)